fr/orm/saving-data.rst

```diff
@@ -223,6 +223,17 @@ de validation utilisé par association::
         ]
     ]);
 
+Le diagramme suivant donne un aperçu de ce qui se passe à l'intérieur
+de la méthode ``newEntity()`` ou ``patchEntity()``:
+
+.. figure:: /_static/img/validation-cycle.png
+   :align: left
+   :alt: Logigramme montrant le process de conversionen entity/validation.
+
+Vous récupérerez toujours une entity en retour de ``newEntity()``. Si la
+validation échoue, votre entité contiendra des erreurs et tous les champs
+invalides seront absents l'entité créée.
```

Comment:
absents **de**

And I'd stick with **entity** in english (there are a couple that are translated)